src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/Elite2ApiHealth.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.justice.digital.hmpps.whereabouts.services;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.boot.actuate.health.HealthIndicator;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+
+@Component
+public class Elite2ApiHealth implements HealthIndicator {
+
+    private final RestTemplate restTemplate;
+
+    @Autowired
+    public Elite2ApiHealth(@Qualifier("elite2ApiHealthRestTemplate") final RestTemplate restTemplate) {
+        this.restTemplate = restTemplate;
+    }
+
+    @Override
+    public Health health() {
+        try {
+            final var responseEntity = this.restTemplate.getForEntity("/health", String.class);
+            return health(responseEntity.getStatusCode());
+        } catch (final RestClientException e) {
+            return health(HttpStatus.SERVICE_UNAVAILABLE);
+        }
+    }
+
+    private Health health(final HttpStatus code) {
+        return health (
+                Health.up(),
```

Comment:
Should this inspect the type of the status to determine whether it's healthy or not?

Reply:
Not sure, It's the code we use in all our backends. Will investigate.